### PR TITLE
Added some missing sqls from Shin's commits.

### PR DESCRIPTION
**Pull request by CookieMonsterDev**
Added some missing sqls from Shin's commits. This will resolve Your Database is not Up to date issues

All this does it add the missing SQL files for Shin's changes.

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1550009247084015800.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+-- Originally from Shin
+-- creature_template https://github.com/azerothcore/azerothcore-wotlk/pull/1322/files
+ALTER TABLE `creature_template`
+  CHANGE `baseattacktime` `BaseAttackTime` INT(10) UNSIGNED DEFAULT 0  NOT NULL,
+  CHANGE `rangeattacktime` `RangeAttackTime` INT(10) UNSIGNED DEFAULT 0  NOT NULL,
+  CHANGE `Health_mod` `HealthModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `Mana_mod` `ManaModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `Armor_mod` `ArmorModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `dmg_multiplier` `DamageModifier` FLOAT DEFAULT 1  NOT NULL,
+  CHANGE `VerifiedBuild` `VerifiedBuild` SMALLINT(5) DEFAULT '0';
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1550009247084015800.sql]: this change is already included here: https://github.com/azerothcore/azerothcore-wotlk/blob/master/data/sql/updates/db_world/2019_01_20_02.sql

Comment (CookieMonsterDev) [reply data/sql/updates/pending_db_world/rev_1550009247084015800.sql]: Mhh it was still giving me an error about it. After a clean dB install and clean pull.

File: data/sql/updates/pending_db_world/rev_1550009247084015800.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  
+-- Smart_Scripts https://github.com/azerothcore/azerothcore-wotlk/pull/1324/files
+ALTER TABLE `smart_scripts`
+  ADD COLUMN `event_param5` INT(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `event_param4`,
+  ADD COLUMN `target_param4` INT(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `target_param3`;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1550009247084015800.sql]: https://github.com/azerothcore/azerothcore-wotlk/blob/master/data/sql/updates/db_world/2019_01_20_00.sql

File: data/sql/updates/pending_db_world/rev_1550009247084015800.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    `entry` mediumint(8) UNSIGNED NOT NULL DEFAULT '0',
+    `faction` smallint(5) unsigned NOT NULL DEFAULT '0',
+    `flags` int(10) unsigned NOT NULL DEFAULT '0',
+    `mingold` mediumint(8) UNSIGNED NOT NULL DEFAULT '0',
+    `maxgold` mediumint(8) UNSIGNED NOT NULL DEFAULT '0',
+    PRIMARY KEY (`entry`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+INSERT INTO `gameobject_template_addon` (`entry`, `faction`, `flags`)
+SELECT `entry`, `faction`, `flags` FROM `gameobject_template`;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1550009247084015800.sql]: https://github.com/azerothcore/azerothcore-wotlk/blob/master/data/sql/updates/db_world/2019_01_28_02.sql

**Comment by CookieMonsterDev** [comment]
We can close this.

**Pull request closed without merging**